geniza/pages/image_formats.py

```diff
@@ -0,0 +1,26 @@
+from django.utils.html import format_html
+from wagtail.images.formats import (
+    Format,
+    register_image_format,
+    unregister_image_format,
+)
+
+
+class CaptionedImageFormat(Format):
+    def image_to_html(self, image, alt_text, extra_attributes=None):
+
+        default_html = super().image_to_html(image, alt_text, extra_attributes)
+
+        return format_html(
+            "<figure>{}<figcaption>{}</figcaption></figure>", default_html, alt_text
+        )
+
+
+unregister_image_format("fullwidth")
+unregister_image_format("left")
+unregister_image_format("right")
```

Comment:
why do these need to be unregistered?

Reply:
My thinking was that these should be unregistered until / unless we have specific styles for them from @gissoo, so that content authors can only pick from those that have been designed. On the designs so far I only see centered images with captions. @gissoo did you want to include styles for left and right aligned images, and/or centered without captions? Or was the intention that images in content pages should only be "centered with captions"?

Reply:
ok, makes sense! those are out of scope for MVP; I just didn't know if they already existed in wagtail (I thought we made them up when we implemented this before). maybe worth a comment?

Reply:
@blms I don't think we discovered a need for left and right images for these pages. I think I originally had them but when we decided to not have a editorial section we agreed to not have left and right images anymore, there doesn't seem to be a need.